src/lib/rest/rest.cpp

```diff
@@ -193,6 +193,21 @@ static int uriArgumentGet(void* cbDataP, MHD_ValueKind kind, const char* ckey, c
 
   LM_T(LmtUriParams, ("URI parameter:   %s: %s", key.c_str(), ciP->uriParam[key].c_str()));
 
+  //
+  // Now check the URI param has no invalid characters
+  // Except for the URI param 'q' that is not to be checked for invalid characters
+  //
+  if (key != "q")
+  {
+    if (forbiddenChars(ckey) || forbiddenChars(val))
```

Comment:
This PR implement ilegal chracters checking in URI params in general. Thus, I think it deserves a line in CNR file (not tied to any specific issue).